QPolygonTransform_lib/src/Transformation.cpp

```diff
@@ -11,14 +12,19 @@ Transform::Transform(const QPointF& rotCenter, float deg_angle)
 
```

Comment:
Extract conversion function to separate function. It may be `constexpr`

Reply:
ok